src/signals/incident-management/containers/IncidentDetail/components/EmailPreview/styled.ts

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+import { themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+import FormFooter from 'components/FormFooter'
```

Comment:
ES lint error

Reply:
This lint error was already there. I've set it to ignore.